Library/ViewModels/PledgeViewModel.swift

```diff
@@ -665,38 +667,44 @@ private func requiresSCA(_ envelope: StripeSCARequiring) -> Bool {
   return envelope.requiresSCAFlow
 }
 
-private func attributedConfirmationString(with project: Project) -> NSAttributedString? {
-  let string = Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_on_project_deadline(
-    project_deadline: Format.date(
-      secondsInUTC: project.dates.deadline,
-      template: "MMMM d, yyyy"
-    )
+private func attributedConfirmationString(with project: Project, pledgeTotal: Double) -> NSAttributedString? {
+  var fullString = ""
+  let date = Format.date(
+    secondsInUTC: project.dates.deadline,
+    template: "MMMM d, yyyy"
   )
+  let pledgeTotal = Format.currency(pledgeTotal, country: project.country)
 
-  guard let attributedString = try? NSMutableAttributedString(
-    data: Data(string.utf8),
-    options: [
-      .documentType: NSAttributedString.DocumentType.html,
-      .characterEncoding: String.Encoding.utf8.rawValue
-    ],
-    documentAttributes: nil
-  ) else { return nil }
+  if project.stats.currentCurrency == project.stats.currency {
+    fullString = Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_on_project_deadline(
+      project_deadline: date
+    )
+  } else {
+    fullString =
+      Strings.If_the_project_reaches_its_funding_goal_you_will_be_charged_total_on_project_deadline(
+        total: pledgeTotal,
+        project_deadline: date
+      )
+  }
+
+  let attributedString: NSMutableAttributedString = NSMutableAttributedString.init(string: fullString)
```

Comment:
For consistency, the syntax without `.init` would be preferred `(NSMutableAttributedString(string: fullString))`